packages/react-ui/src/features/flows/components/import-flow-dialog.tsx

```diff
@@ -106,7 +164,10 @@ const ImportFlowDialog = ({
           <Button onClick={handleSubmit} loading={isPending}>
             {t('Import')}
           </Button>
-        </DialogFooter>
+        </div>
+        {errorMessage && (
+          <div className={cn(formMessageClass, 'mt-4')}>{errorMessage}</div>
```

Comment:
Why not use ErrorMessage? or inline the formMessageClass if it not create a component or use `ErrorMessage`?

Reply:
Answered in a previous comment